Adyen.EcommLibrary.Test/BaseTest.cs

```diff
@@ -25,17 +26,17 @@ public PaymentResult CreatePaymentResultFromFile(string fileName)
             var paymentResult = payment.Authorise(paymentRequest);
             return GetAdditionaData(paymentResult);
         }
-       
+
         #endregion
-        
+
         #region Modification objects
 
         protected CaptureRequest CreateCaptureTestRequest(string pspReference)
         {
             var captureRequest = new CaptureRequest
             {
                 MerchantAccount = "MerchantAccount",
-                ModificationAmount = new Amount("EUR", 150),
+                ModificationAmount = new Model.Amount("EUR", 150),

```

Comment:
Amount seems better then Model.Amount.

Reply:
Again the reason is now there are two Amount classes. We can add 
using PaymentResult = Adyen.EcommLibrary.Model.Amount;